Tasks/AzureRmWebAppDeployment/task.loc.json

```diff
@@ -17,7 +17,7 @@
   "version": {
     "Major": 4,
     "Minor": 0,
-    "Patch": 5
```

Comment:
increase minor version instead of patch version.

Reply:
done